vars/enforceTrustedApproval.groovy

```diff
@@ -37,12 +61,14 @@ final boolean isOrgMember(String user, String org, String gitHubCredentialsId) {
 def call(String trustedOrg='fheng', String gitHubCredentialsId='githubjenkins') {
     if (!env.CHANGE_AUTHOR) {
         println "This doesn't look like a GitHub PR, continuing"
-    } else if (!isOrgMember(env.CHANGE_AUTHOR, trustedOrg, gitHubCredentialsId)) {
+    } else if (isTrustedUser(env.CHANGE_AUTHOR, gitHubCredentialsId)) {
+        println "${env.CHANGE_AUTHOR} is trusted, continuing"
+    } else if(isOrgMember(env.CHANGE_AUTHOR, trustedOrg, gitHubCredentialsId)) {
+        println "${env.CHANGE_AUTHOR} is trusted, continuing"
```

Comment:
Let's differantiate the string output here, one should say `..is trusted as a member of the Candlepin org, continuing`, and the other `..is trusted as an external contributor, continuing`

Reply:
Done.